rllib/tests/test_dqn_internal_custom_model.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+import ray
+from ray.rllib.agents.dqn import DQNTrainer
+from ray.rllib.agents.ppo import PPOTrainer
+from ray.rllib.models import ModelCatalog
+from ray.rllib.models.tf.visionnet_v2 import VisionNetwork
+from ray.rllib.models.tf.fcnet_v2 import FullyConnectedNetwork
+
+class DQNCustomModelTest(unittest.TestCase):
```

Comment:
Instead of adding a new test, can you add an case to https://github.com/ray-project/ray/blob/master/rllib/examples/custom_keras_model.py exercising this instead?